packages/@dcos/sorting/index.ts

```diff
@@ -0,0 +1,61 @@
+export enum ComparisonResult {
+  BIGGER = 1,
+  EQUAL = 0,
+  SMALLER = -1
+}
+export type Comparator<T> = (a: T, b: T) => ComparisonResult;
+export type Lense<T> = (data: T) => any;
```

Comment:
I really like the incorporation of lenses to zoom in the particular attribute to be sorted.

Reply:
🔡  _Lens_ 😅